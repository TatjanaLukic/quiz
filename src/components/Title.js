import React from "react";

export const Title = props => {
  return (
    <h1
      style={{
        color: "#8f34eb",
        background: "white",
        marginTop: "30px",
        textAlign: "center",
        fontSize: "72px",
        marginBottom: "40px",
        padding: "30px 0px"
      }}
    >
      {props.title}
    </h1>
  );
};

export default Title;
