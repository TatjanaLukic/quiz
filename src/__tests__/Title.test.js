import React from "react";

import { configure, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

import Title from "../components/Title";

configure({ adapter: new Adapter() });

describe("<Title/>", () => {

  const title = "test title";

  it("spanshot matches" , () => {
    const tree = shallow(<Title title />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});
