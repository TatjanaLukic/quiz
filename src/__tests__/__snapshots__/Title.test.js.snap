// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title/> spanshot matches 1`] = `
<h1
  className="Title"
>
  <Component />
</h1>
`;
